metrics/Exposure_metrics/histogram_metrics_driver_script.py

```diff
@@ -0,0 +1,35 @@
+from histogram_metrics import histogram_mertics
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.